### ISSUE TITLE
Dump OBJECTS_OBJREF_NOT_ASSIGNED_NO when importing interfaces

### ISSUE THREAD

**Issue by pokrakam** [abapGit/abapGit#6016]
Importing INTF dumps, possibly due to pushing/pulling abapGit version differences. Looking into this.

**Comment by pokrakam** [comment]
It is due to mixed json and xml metadata. This piece of code assumes no XML if it finds any `json` files. Question is whether this is intentional design? Issue message or mixed handling?
https://github.com/abapGit/abapGit/blob/0eb41621833c226e767f8894fb567f03a208e48e/src/objects/zcl_abapgit_objects.clas.abap#L690-L697

**Comment by larshp** [comment]
if you have some JSON files for metadata, it happens when experimental features are enabled

suggest disabling experimental features

**Comment by pokrakam** [comment]
Yes, they're on. So somewhere `INTF` was defaulted to AFF, then back again. Solution is easy - push repo again with XML.

**Comment by mbtools** [comment]
You got it. There was a push when experimental features was turned on.
